src/main/daml/ContingentClaims/Core/Claim.daml

```diff
@@ -70,6 +71,11 @@ or (_, Or a b cs) (tag2, claim) = Or a b (cs ++ [(tag2, claim)])
 or (tag1, claim) (_, (Or a b cs)) = Or (tag1, claim) a (b :: cs)
 or (tag1, claim) (tag2, claim') = Or (tag1, claim) (tag2, claim') []
 
+-- | Smart constructor for a list of claims to which `Or` should be applied.
```

Comment:
For consistency, we could also define `ands = mconcat`

Reply:
I tried that first, but there was a name clash from the internal CC.

Reply:
> For consistency, we could also define `ands = mconcat`

I opened a separate issue for this: https://github.com/digital-asset/daml-finance/issues/698
I did not want to add it without actually using it or adding proper tests, to ensure that it works as expected.

Reply:
> > What about ors as a name to the function? It would fit nicely with the already existing or
> 
> I tried that first, but there was a name clash from the internal CC.

Makes sense, the `ors` is now one of the record fields on the tree.